### PR TITLE
Feat: display quote to home screen

### LINKED ISSUE #8: Modal- Quote screen
- Quote screen
- Should hold state
- Should be directed to quote screen from greeting button 
- Should be able to favorite a quote or advice
- Should use information from API's

### PR DESCRIPTION
**Pull request by GOECHA**
# Pull Request
Display API data to home screen. To GreetingButton! 

- [x] New feature (adds functionality)
- [x] Describe the purpose of this PR(other):
- [x] I have added screenshots, when necessary (see below):

# Please describe the tests that you ran to verify your changes.

- [x] I tested my changes in the simulator
- [ ] I tested with Cypress

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any other comments, questions or concerns?

Screen Shot 2022-09-14 at 7 54 29 PM

### CODE REVIEW

File: components/GreetingButton.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { View, Text, StyleSheet, TouchableHighlight } from 'react-native';
 
 
-const GreetingButton = ({mantras}) => {
+const GreetingButton = ({ quote, fetch}) => {
```
Comment (Maia-Cochran) [inline components/GreetingButton.js:4]: Passing params in here... I believe I'm following how this is working, but would love to go through a code review together for understanding :)

File: components/GreetingButton.js [hunk 2]
```diff
@@ -10,9 +10,9 @@ const GreetingButton = ({mantras}) => {
 
 
     return (
-       <TouchableHighlight onPress={mantras}>     
+       <TouchableHighlight onPress={fetch}>
```
Comment (Maia-Cochran) [inline components/GreetingButton.js:13]: TouchableHighlight works similar to Pressable? Very cool!

File: components/Home.js [hunk 3]
```diff
@@ -4,34 +4,38 @@ import GreetingButton from './GreetingButton'
...
+
+
     return (
         <View style={styles.homeContainer}>
             <Header />
-            <TouchableHighlight onPress={mantras}>  
-              <GreetingButton text={this.state.text}/>
-            </TouchableHighlight>
+            {/* <TouchableHighlight onPress={mantras}>   */}
+              <GreetingButton quote={quote.mantra} fetch={fetchApiCall}/>
```
Comment (Maia-Cochran) [inline components/Home.js:37]: Ahhh, this is making more sense. Sweet!

File: components/mantraInterval.js [hunk 4]
```diff
@@ -1,47 +1,47 @@
...
+//   const [loading, setLoading] = useState(true);
 
-  const fetchData = async () => {
-    const resp = await fetch("http://localhost:3001/mantra");
-    const data = await resp.json();
-    setData(data);
-    setLoading(false);
-  };
+//   const fetchData = async () => {
+//     const resp = await fetch("http://localhost:3001/mantra");
```
Comment (Maia-Cochran) [inline components/mantraInterval.js:9]: Awesome work building out this fetch.

File: components/mantraInterval.js [hunk 5]
```diff
@@ -1,47 +1,47 @@
...
-        {item}
-      </Box>
-    );
-  };
-  useEffect(() => {
-    fetchData();
-    const dataInterval = setInterval(() => fetchData(), 5 * 1000);
+//   const renderItem = ({ item }) => {
+//     return (
+//       <Box px={5} py={2} rounded="md" bg="primary.300" my={2}>
```
Comment (Maia-Cochran) [inline components/mantraInterval.js:17]: Is this styling in a way?

File: components/mantraInterval.js [hunk 6]
```diff
@@ -1,47 +1,47 @@
...
-    return () => clearInterval(dataInterval);
-  }, []);
+//     return () => clearInterval(dataInterval);
+//   }, []);
 
 
-  return (
-    <NativeBaseProvider>
+//   return (
+//     <NativeBaseProvider>
```
Comment (Maia-Cochran) [inline components/mantraInterval.js:31]: NativeBaseProvider? I'll have to look into this. LOVE how much we're learning in this project!

**Review by Maia-Cochran — APPROVED** [review]
Outstanding work, Chantal! I can't wait to review this and study up on what you learned! Totally crushing it.

**Review by DrSLMac — APPROVED** [review]
Sweet!!

**Pull request merged**